contributing/code_of_conduct/reporting_guidelines.rst

```diff
@@ -48,17 +50,16 @@ and that everyone is physically safe.
 Once the team has a complete account of the events, they will make a decision as
 to how to respond. Responses may include:
 
-  * Nothing (if we determine no code of conduct violation occurred).
+  * Nothing (if we determine no Code of Conduct violation occurred).
   * A private reprimand from the Code of Conduct response team to the individual(s)
     involved.
-  * A public reprimand.
   * An imposed vacation (i.e. asking someone to "take a week off" from a mailing
     list or Slack).
   * A permanent or temporary ban from some or all Symfony conference/community
     spaces (events, meetings, mailing lists, IRC, Slack, etc.)
   * A request to engage in mediation and/or an accountability plan.
-  * On a case by case basis, the enforcement team might share information an incidents
-    with employers or other 3rd parties in order to fulfill their mandate.
+  * On a case by case basis, other actions may be possible but will usually be
```

Comment:
`...but will usually be coordinated...` -> `...but will always be coordinated...` ?

Reply:
I think we said “usually” because we expect for extreme cases for there to be coordination but for trivial cases that might require a different response it might not make sense. But I if this is too much uncertainity to be acceptable, then I would be ok with making it “always”.